package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@slack/client": "^3.16.0",
+    "@slack/web-api": "^5.14.0",
```

Comment:
The old SDK used before did not support sending messages with the new block format. I added the new Slack SDK as a dependency just for the new Slack API methods we need. I'm keeping the old SDK as well until we get a chance to clean the old one up.